dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/client/Client.java

```diff
@@ -40,10 +46,10 @@ public ProviderAuthenticationResult continueFlow(HttpServletRequest request, Htt
         Flow flow = AbstractFlow.getInstance(oidcClientConfig);
         return flow.continueFlow(request, response);
     }
-    
-    public void validate(TokenResponse tokenResponse) throws TokenValidationException {
+
+    public JwtClaims validate(TokenResponse tokenResponse) throws TokenValidationException {
```

Comment:
Adding org.jose4j.jwt.JwtClaims caused a build failure at at io.openliberty.security.jakartasec.cdi.beans.OidcHttpAuthenticationMechanismTest.setUp(OidcHttpAuthenticationMechanismTest.java:90), where the Client class cannot be mocked due to a ClassNotFoundException:

Caused by: java.lang.ClassNotFoundException: org.jose4j.jwt.JwtClaims

The testpath in bnd.bnd for the io.openliberty.security.jakartasec.cdi will need to be updated to include jose4j.